Documentation/ceph-examples.md

```diff
@@ -13,12 +13,12 @@ indent: true
 
 # Ceph Examples
 
-Configuration for Rook and Ceph can be configured in multiple ways to provide block devices, shared file system volumes or object storage in a kubernetes namespace. We have provided several examples to simplify storage setup, but remember there are many tunables and you will need to decide what settings work for your use case and environment. 
+Configuration for Rook and Ceph can be configured in multiple ways to provide block devices, shared file system volumes or object storage in a kubernetes namespace. We have provided several examples to simplify storage setup, but remember there are many tunables and you will need to decide what settings work for your use case and environment.
```

Comment:
The object storage is not coming through the CSI driver. Do we need to mention it here?